test/units/testsuite-15.sh

```diff
@@ -124,6 +124,32 @@ EOF
     clear_services test15-a test15-b test15-c
 }
 
+test_linked_units () {
+    echo "Testing linked units..."
+    echo "*** test linked unit (same basename)"
+
+    create_service test15-a
+    mv /etc/systemd/system/test15-a.service /
+    ln -s /test15-a.service /etc/systemd/system/
+    ln -s test15-a.service /etc/systemd/system/test15-b.service
+
+    check_ok test15-a Names test15-a.service
+    check_ok test15-a Names test15-b.service
+
+    echo "*** test linked unit (cross basename)"
+
+    mv /test15-a.service /test15-a@.scope
+    ln -fs /test15-a@.scope /etc/systemd/system/test15-a.service
+    systemctl daemon-reload
+
+    check_ok test15-a Names test15-a.service
+    check_ok test15-a Names test15-b.service
+    check_ko test15-a Names test15-b@
```

Comment:
Was this supposed to be `test15-a@` and not `test15-b@`? I'm puzzled as to why `test15-b@` would have been a possible option

Reply:
I'm not sure any more what I had in mind ;) But yeah, `test15-@` would make more sense.

Reply:
https://github.com/systemd/systemd/pull/18911